services/requests/feasibility_constraint_definitions/schemas.go

```diff
@@ -0,0 +1,120 @@
+package feasibility_constraint_definitions
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/leanspace/terraform-provider-leanspace/helper"
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+)
+
+var feasibilityConstraintDefinitionSchema = map[string]*schema.Schema{
+	"id": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"name": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"description": {
+		Type:     schema.TypeString,
+		Optional: true,
+	},
+	"cloned": {
+		Type:     schema.TypeBool,
+		Optional: true,
```

Comment:
Are you sure it's not computed? Also I don't see it in the API docs

Reply:
Yeah, I looked at the code and did not see the `@JsonIgnore` on it

Reply:
Ok then could you check if either it is exposed and then it should be present in the API doc or if it shouldn't be exposed and then add this `@JsonIgnore` annotation?

Reply:
it is not exposed, you were right pointing this out  :)